app/helpers/green_lanes_helper.rb

```diff
@@ -63,6 +63,10 @@ def yes_no_not_sure_options
     [OpenStruct.new(id: 'yes', name: 'Yes'), OpenStruct.new(id: 'no', name: 'No'), OpenStruct.new(id: 'not_sure', name: 'Not sure')]
   end
 
+  def green_lanes_eligibility_start_path
```

Comment:
Just thought, do we start changing the method name now? something like;

`def spimm_eligibility_start_path`

Reply:
good point, but it's probably better to create a ticket to rename all of those,
there are 267 occurrences of 'green lanes' !
so I'd avoid to create mixed notation